app/src/main/java/edu/grinnell/grinnell_publications_android/Fragments/PublicationsFragment.java

```diff
@@ -3,20 +3,33 @@
 
 import android.os.Bundle;
 import android.support.v4.app.Fragment;
+import android.support.v4.view.MenuItemCompat;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.support.v7.widget.SearchView;
 import android.view.LayoutInflater;
 import android.view.Menu;
 import android.view.MenuInflater;
+import android.view.MenuItem;
 import android.view.View;
 import android.view.ViewGroup;
 
+import java.util.ArrayList;
+import java.util.List;
+
 import edu.grinnell.grinnell_publications_android.Models.Interfaces.UserInterface;
+import edu.grinnell.grinnell_publications_android.Models.Realm.RealmPublication;
 import edu.grinnell.grinnell_publications_android.R;
 
 /**
  * {@link Fragment} to display users' subscribed publications.
  * @author Larry Boateng Asante
  */
-public class PublicationsFragment extends Fragment implements UserInterface {
+public class PublicationsFragment extends Fragment implements UserInterface, SearchView.OnQueryTextListener {
+
+    RecyclerView recyclerView;
```

Comment:
Please follow the variable naming convention here. Also, you should make this `private`. Access modifiers matter :)